### DISCUSSION TITLE
npm adduser  500 Internal Server Error

### DISCUSSION THREAD

**Discussion by nicky132** [verdaccio/verdaccio#3498] [Q&A] [answered]
**Your Environment**
  
 * **verdaccio version**: Docker version 20.10.21, build baeda1f
 * **node version** [12.x.x, 14.x.x]:[v16.16.0]
 * **package manager**: [npm@8, yarn@1]
 * **os**: [centos7]
 * **platform**: [docker]

**Describe the bug**

![1668407655660_D14AB62E-233E-4a53-91E2-28FE5AF1F069](https://user-images.githubusercontent.com/34881461/201591185-28c35738-567f-4503-b1bd-5b987c032cb7.png)

**To Reproduce**

**Expected behavior**

**Screenshots, server logs, package manager log**

**Configuration File (cat ~/.config/verdaccio/config.yaml)**
`storage: /verdaccio/storage
plugins: /verdaccio/plugins

web:
  title: xxxx

publish:
  allow_offline: true
auth:
  htpasswd:
    file: /verdaccio/conf/htpasswd

uplinks:
  taobao:
    url: https://registry.npm.taobao.org
    timeout: 3000ms
  yarnpkg:
    url: https://registry.yarnpkg.com/
    timeout: 6000ms
  npmjs:
    url: https://registry.npmjs.org/
    timeout: 6000ms
  local:
    url: http://10.100.218.64:4873/
    timeout: 6000ms
packages:
  '@*/*':  
    access: $all
    publish: superAdmin cico admin
    unpublish: superAdmin cico admin
    proxy: taobao yarnpkg npmjs local

  '**':
    access: $all

    publish: superAdmin cico admin
    unpublish: superAdmin cico admin

    proxy: taobao yarnpkg npmjs local

server:
  keepAliveTimeout: 60
middlewares:
  audit:
    enabled: true

logs: { type: stdout, format: pretty, level: http }
max_body_size: 100mb # 设置最大包体积
listen:
  - 10.100.218.64:4873
i18n:
  web: zh-CN`

**Environment information**

**Debugging output**

- `$ NODE_DEBUG=request verdaccio` display request calls (verdaccio <--> uplinks)
- `$ DEBUG=verdaccio* verdaccio` enable extreme verdaccio debug mode (verdaccio api)
- `$ npm -ddd` prints:
- `$ npm config get registry` prints:

**Contribute to Verdaccio**

- [ ] I'm willing to fix this bug 🥇

**Comment by juanpicado** [comment, marked as answer]
Please 🙏🏼 , only create issues if you prove it could be an issue. https://verdaccio.org/community/contributing#whats-is-not-considered-a-bug otherwise create a discussion and I'll be happy to answer it.

If there is 500 error then the answer probably is on the server logs in your docker image.

**Reply by nicky132** [reply]
already solve the problem by
chmod  -R  777  verdaccio : ) 💯

**Reply by karthiksedoc** [reply]
> already solve the problem by chmod -R 777 verdaccio : ) 💯

I changed the ownership of the folder in to the user who's running the process, thought the idea of opening the folders to world is a bad idea
